.prettierrc

```diff
@@ -0,0 +1,12 @@
+{
+    "trailingComma": "es5",
+    "tabWidth": 4,
+    "semi": true,
+    "singleQuote": false,
```

Comment:
Unless you have a strong opinion for double quotes, I think I'd prefer single quotes. It would make the diff smaller, as I tend to lean towards single quotes anyways. But I could be talked out of it.

Reply:
No problems here!